src/veh_interact.cpp

```diff
@@ -375,12 +375,7 @@ void veh_interact::cache_tool_availability()
 
     has_wrench = crafting_inv.has_quality( quality_id( "WRENCH" ) );
 
-    has_wheel = crafting_inv.has_components( "wheel", 1 ) ||
-                crafting_inv.has_components( "wheel_wide", 1 ) ||
-                crafting_inv.has_components( "wheel_armor", 1 ) ||
-                crafting_inv.has_components( "wheel_bicycle", 1 ) ||
-                crafting_inv.has_components( "wheel_motorbike", 1 ) ||
-                crafting_inv.has_components( "wheel_small", 1 );
+    has_wheel = crafting_inv.has_components( "any", 1, []( const item & it ){ return item::find_type( it.typeId() )->wheel ? true : false; } );
```

Comment:
```suggestion
    has_wheel = crafting_inv.has_components( "any", 1, []( const item & it ){ return item::find_type( it.typeId() )->wheel; } );
```

Reply:
the return type would then be `cata::optional<islot_wheel>` instead of bool.  It doesn't want to implicitly cast for some reason in this case.

Reply:
!! it then?

Reply:
Oh yeah that would be nicer.